Servers/UI/OJS.Servers.Ui/ClientApp/src/components/submissions/test-run-details/TestRunDetails.tsx

```diff
@@ -69,11 +92,13 @@ const TestRunDetails = ({ testRun }: ITestRunDetailsProps) => {
                 />
                 <span>
                     {testRun.memoryUsed}
-                    MB
                 </span>
             </span>
+            <span className={styles.testRunDataParagraph}>
+                {renderLabel()}
```

Comment:
spaces after {    and     }

Reply:
Not sure what you meant. I hope ESlint fixed it though.